README.md

```diff
@@ -41,14 +41,14 @@ NZXT Kraken X (X42, X52, X62 or X72)
 ├── Liquid temperature     31.7  °C
 ├── Fan speed               801  rpm
 ├── Pump speed             2239  rpm
-└── Firmware version      6.0.2  
```

Comment:
This change is fine, but FYI the actual output has those two trailing spaces when the status item has no unit.